workers/unity/Packages/io.improbable.gdk.core/Subscriptions/EntitySubscriptions.cs

```diff
@@ -1,36 +0,0 @@
-using System;
-using Improbable.Gdk.Core;
-using Unity.Entities;
-
-namespace Improbable.Gdk.Subscriptions
-{
-    public static class EntitySubscriptions
```

Comment:
what was the initial purpose of this class?

Reply:
It has never been used.